airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/utils/common.py

```diff
@@ -59,6 +60,18 @@ def filter_output(records: Iterable[AirbyteMessage], type_) -> List[AirbyteMessa
     return list(filter(lambda x: x.type == type_, records))
 
 
+def remove_ignored_fields(output: List[AirbyteMessage], ignored_fields: Mapping[str, List[str]]) -> List[Mapping[str, Any]]:
```

Comment:
this function does too much and doesn't reflect its name.
how about have additional argument in `serialize` function:
```python
def serialize(value, exclude_fields=None):
...
```
then your code will be:
```python
serializer = partial(serialize, exclude_fields= ignored_fields)
output_diff = set(map(serializer, records_1)) - set(map(serializer, records_2))

```